flash/vision/segmentation/transforms.py

```diff
@@ -11,16 +11,26 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-from typing import Callable, Dict, Tuple
+from typing import Any, Callable, Dict, Sequence, Tuple
 
 import kornia as K
 import torch
 import torch.nn as nn
+from torch.utils.data._utils.collate import default_collate
 
 from flash.data.data_source import DefaultDataKeys
 from flash.data.transforms import ApplyToKeys, KorniaParallelTransforms
 
 
+def collate(samples: Sequence[Dict[str, Any]]) -> Dict[str, Any]:
```

Comment:
this is duplicated code in `flash/vision/classification/transforms.py ` , right ?